packages/builder/src/components/nav/ModelNavigator/BlockNavigator.svelte

```diff
@@ -19,6 +19,10 @@
   function addField(blockDefinition) {
     backendUiStore.actions.models.addField(blockDefinition)
     backendUiStore.actions.models.fetch()
+
+    setTimeout(() => {
```

Comment:
same as above - you may need `tick` here, or even just `await` on line 21 could work